.travis.yml

```diff
@@ -1,33 +1,34 @@
+language: python
+
+python:
+  - "2.7"
+
 addons:
   apt:
     packages:
       - expect-dev  # provides unbuffer utility
-      - python-lxml # because pip installation is slow
+      - python-lxml  # because pip installation is slow
 
-language: python
-python:
-  - "2.7"
+env:
+  global:
+  - VERSION="7.0" TESTS="0" LINT_CHECK="0"
```

Comment:
Here https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L14 I see also ```TRANSIFEX="0"```. Is there any reason to have removed it?